dgraph/cmd/alpha/dashboard.go

```diff
@@ -38,8 +38,18 @@ func homeHandler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "404 page not found", http.StatusNotFound)
 		return
 	}
-	x.Check2(w.Write([]byte(
-		"Dgraph browser is available for running separately using the dgraph-ratel binary")))
+
+	dgraph := "\nDgraph is up and running...\n\n" +
+		"Now you can access your cluster via Ratel Dashboard.\n" +
+		"Remember, the PORT to use there to connect to Alpha is 8080 + offset if the case.\n" +
+		"If you are running GraphQL you can use any GraphQL Client from the open source community.\n" +
+		"\nThank you for using Dgraph!\n" +
+		"\nImportant Links:\n\nhttps://dgraph.io/docs\nhttps://discuss.dgraph.io/\nhttps://cloud.dgraph.io/ " +
+		"\n\n\nCopyright 2016-2022 Dgraph Labs, Inc. and Contributors.\n"
```

Comment:
Just a nitpick, is there a reason we don't use a raw string literal here?  i.e.
```
`
Dgraph is up and running...

Now you can access...
Remember....
`
```

Reply:
Ha. See https://github.com/dgraph-io/dgraph/pull/8458#discussion_r1036323692

Reply:
Okay, I dicide to use HTML/CSS. I've share a printscreen in Slack. 
Basically it will use backtick and this won't have any side effects that I was complaning about Plain Text rendering.

```
dgraph := `<body>
	<BR><h2>Dgraph is up and running...
```